index.html

```diff
@@ -68,6 +69,124 @@ <h2>Hey There. I'm Charles.</h2>
+            <p class="works-description">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+            <ul class="competencies">
+              <li>CSS</li>
+              <li>HTML</li>
+              <li>JAVASCRIPT</li>
+            </ul>
+          </div>
+          <div><button class="btn-type-b">See project</button></div>
+        </div>
+        <div class="grid-item">
+          <img src="images/Img Placeholder (1).png" alt="place holder image 2">
+          <div class="image-text">
+            <div><h3 class="bold-text">Profesional Art Printing Data</h3></div>
+            <p class="works-description">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+            <ul class="competencies">
+              <li>CSS</li>
+              <li>HTML</li>
+              <li>JAVASCRIPT</li>
+            </ul>
+          </div>
+          <div><button class="btn-type-b">See project</button></div>
+        </div>
+        <div class="grid-item">
+          <div class="placeholder-image">
+            <img src="images/Img Placeholder (1).png" alt="place holder image 2">
+          </div>
+          <div class="image-text">
+            <div><h3 class="bold-text">Profesional Art Printing Data</h3></div>
+            <div class="works-description">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</div>
+            <ul class="competencies">
+              <li>CSS</li>
+              <li>HTML</li>
+              <li>JAVASCRIPT</li>
+            </ul>
+          </div>
+          <div><button class="btn-type-b">See project</button></div>
+        </div>
+      </div>
+    </section>
+    <section class="about-me-section">
+      <div><h2  class="about-me-title">About me</h2></div>
+      <div><p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p></div>
+      <div><button class="abount-me-btn">Get my resume</button></div>
+      <img class="right-side-image-b" src="images/illustration-about-me-mobile.png" alt="right-design about-me section">
+      <img class="hor-line" src="images/Devider.png" alt="horizental line">
+      <div class="list-container">
+        <ul class="about-me-cat">
+          <li class="language">
+            Languages
+              <div class="lang-list-container">
+                <div class="language-list">Javascript</div>
+                <div class="language-list">CSS</div>
+                <div class="language-list">HTML</div>
+              </div>
+          </li>
+          <li class="frameworks">Frameworks</li>
+          <li class="skills">Skills</li>
+        </ul>
+      </div>
```

Comment:
As the previous reviewer requested please complete this section with all the information provided with Figma template. The goal of this project is to code a given design pixel-perfect. It needs to be an exact copy of Figma template. Kindly ensure, you added all languages, frameworks and skills on the template.
<img width="335" alt="Screenshot 2023-03-25 at 12 37 31 AM" src="https://user-images.githubusercontent.com/25887253/227646859-7f9a64c8-f4fe-4252-b463-a6c297303d2a.png">